source/system/String.ooc

```diff
@@ -26,6 +26,7 @@ String: class {
 		this _buffer free()
 		super()
 	}
+	safeFree: func { if (!this _isLiteral) this free() } // Will not free literals automatically
```

Comment:
We could probably block the regular free using a hidden lock flag but explicitly freeing literals is bad style if it does nothing and there were leaks last time I tried using a lock.